### ISSUE TITLE
aria-multiline attribute on Autocomplete

### ISSUE THREAD

**Issue by vertigo7x** [angular/material2#7100]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Pass wai-aria validation using Axe

#### What is the current behavior?
The autocomplete component have an input with an attribute aria-multiline set to false, but the specification [wai-aria-1.1](https://www.w3.org/TR/wai-aria-1.1/#combobox) says that the child element should have the aria-multiline attribute: 

> A composite widget containing a single-line textbox and another element, such as a listbox or grid, that can dynamically pop up to help the user set the value of the textbox. 
> Authors must ensure an element with role combobox contains or owns a text input element with role textbox or searchbox and that the text input has aria-multiline set to false. If the combobox provides autocompletion behavior for the text input as described in aria-autocomplete, authors must set aria-autocomplete on the textbox element to the value that corresponds to the provided behavior.

#### What are the steps to reproduce?
Pass accessibility validation using Axe for Chrome on [Autocomplete](https://material.angular.io/components/autocomplete/examples)

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
material 2.0.0-beta.10
